mmv1/products/compute/api.yaml

```diff
@@ -8395,9 +8395,20 @@ objects:
       - !ruby/object:Api::Type::Integer
         name: 'size'
         description: |
-          The total number of nodes in the node group.
-        required: true
+          The total number of nodes in the node group. One of `initial_size` or `size` must be specified.
```

Comment:
You can edit the description in `terraform.yaml` too so that they don't refer to a field that doesn't exist.